tests/spark/test_spark_model_export.py

```diff
@@ -58,11 +58,11 @@ def spark_custom_env(tmpdir):
 @pytest.fixture(scope="session", autouse=True)
 def spark_context():
     conf = pyspark.SparkConf()
-    conf.set(
-        key="spark.jars.packages",
-        value="ml.combust.mleap:mleap-spark-base_2.11:0.12.0,"
-        "ml.combust.mleap:mleap-spark_2.11:0.12.0",
-    )
+    # conf.set(
+    #     key="spark.jars.packages",
+    #     value="ml.combust.mleap:mleap-spark-base_2.11:0.12.0,"
+    #     "ml.combust.mleap:mleap-spark_2.11:0.12.0",
+    # )
```

Comment:
This seems to be the root cause of the failure: https://github.com/mlflow/mlflow/runs/2388530643

Reply:
Opened a PR to address this issue: https://github.com/mlflow/mlflow/pull/4268